setup.py

```diff
@@ -49,6 +49,13 @@ def read(*rnames):
       package_dir={'': 'src'},
       install_requires=[
           'setuptools',
+
+      ],
+      setup_requires=[
+          'pytest-runner',
```

Comment:
@loechel To my understanding buildout does something like `python setup.py install` so it will install `pytest-runner` in projects where `RestrictedPython` is a dependency. (I have seen this behaviour on other packages which used `setup_requires`.)

@stephan-hof https://pypi.python.org/pypi/pytest-runner#conditional-requirement looks promising but I did not yet use it anywhere.